styles.css

```diff
@@ -0,0 +1,535 @@
+}
+
+.uppercase {
+    text-transform: uppercase;
+}
+
+.lowercase {
+    text-transform: lowercase;
+}
+
+button {
+    margin: 40px;
+    border: 0;
+    border-radius: 50vh;
+    padding: 20px 40px 20px 40px;
+    box-shadow: 5px 5px 15px var(--shadow);
+
+    display: flex;
+    flex-direction: column;
+    justify-content: stretch;
+    flex-grow: 1;
+    align-items: center;
+}
+
+section.work span.image-wrapper img {
+    box-shadow: 5px 5px 35px var(--black);
+    border-radius: 16px;
+}
+
+.image-wrapper {
+    flex-grow: 0;
+    flex-shrink: 1;
+}
+
+.image-wrapper img {
+    width: 100%;
+    object-fit: cover;
+}
+
+section.services article {
+    border-style: solid;
+    border-width: 2px;
+    box-shadow: 5px 5px 15px var(--shadow);
+
+}
+
+input, textarea {
+    margin: 5px 0;
+    border: 0;
+    padding: 10px;
+    resize: vertical;
+}
+
+input:focus-visible, textarea:focus-visible {
+    outline: none;
+}
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+
+header {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+header nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+}
+
+header nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-grow: 1;
+    padding: 20px;
+}
+
+header section {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    padding: 0 40px;
+}
+
+header section article {
+    max-width: 700px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    padding: 80px 0;
+}
+
+section.header article h1, section.header article h2, section.header article p
+{
+    text-align: center;
+    padding: 0.6em 0;
+}
+
+
+
+
+header {
+    position: relative;
+    padding-bottom: 150px;
+}
+
+span.squiggle > img {
+    position: absolute;
+    bottom: 0;
+    left: calc((100% - 80px) / 2);
+}
+
+/* --------------- 4.2 Main ------------------- */
+
+main {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: stretch;
+}
+
+main section article {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+}
+
+/* --------------- 4.2.1 Hello ------------------- */
+
+section.hello {
+    flex-basis: 501px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    padding: 50px;
+}
+
+section.hello article {
+    max-width: 700px;
+}
+
+section.hello article > * {
+    padding: 1em 0;
+}
+
+section.hello h1, section.hello h2, section.hello p {
+    text-align: center;
+}
+
+/* --------------- 4.2.2 Work ------------------- */
+
+section.work {
+    flex-basis: 502px;
```

Comment:
Ben wel benieuwd waarom precies dit getal? Kwam dit toevallig goed uit op jouw scherm? Wellicht dat andere mensen met een andere resolutie monitor wel iets anders gaan ervaren

Reply:
Haha, goed gespot. En op regel 324 staat ook nog 501px, trouwens.
Dit soort waarden gebruik ik soms tijdens het testen, zodat ik in de Inspector snel kan zien waar een waarde vandaan komt. Vergeten terug te zetten.